roles/aws/create-instance/tasks/dns.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
We have previously create a common set of DNS roles (and playbooks) to work with Route53 to avoid having specific roles handle this aspect, so I'd suggest updating to use the existing DNS playbook (or roles) for this purpose - i.e.: 
Playbooks are found here:
https://github.com/redhat-cop/infra-ansible/blob/master/playbooks/update-dns-zones.yml
https://github.com/redhat-cop/infra-ansible/blob/master/playbooks/update-dns-records.yml

... and more details around the roles can be found here:
https://github.com/redhat-cop/infra-ansible/tree/master/roles/dns